lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/userdata_airloop_hvac.csv

```diff
@@ -0,0 +1 @@
+name,has_fan_power_credit_fully_ducted,has_fan_power_credit_return_or_exhaust_flow_control�,has_fan_power_credit_exhaust_treatment ,has_fan_power_credit_filtration_m9to12,has_fan_power_credit_filtration_m13to15,clean_filter_pressure_drop_for_fan_power_credit_filtration_m16plus,fan_power_credit_gas_phase_cleaners,fan_power_credit_biosafety,fan_power_credit_other_than_coil_runaround,has_fan_power_credit_coil_runaround,fan_power_credit_evaporative_humidifier_or_cooler,has_fan_power_credit_sound_attenuation,has_fan_power_credit_exhaust_serving_fume_hoods,has_fan_power_credit_lab_or_vivarium_highrise_vertical_duct,economizer_exception_for_gas_phase_air_cleaning,economizer_exception_for_open_refrigerated_cases,exhaust_energy_recovery_exception_for_toxic_fumes_etc,exhaust_energy_recovery_exception_for_type1_kitchen_hoods,exhaust_energy_recovery_exception_for_type_distributed_exhaust ,exhaust_energy_recovery_exception_for_dehumidifcation_with_series_cooling_recovery
```

Comment:
Do we need to stage these files? Some of the fields include strange characters.

Reply:
I did a search and hopefully got all of the bad chars.